Source/WebCore/html/HTMLMediaElement.cpp

```diff
@@ -4939,11 +4938,6 @@ void HTMLMediaElement::configureTextTrackGroup(const TrackGroup& group)
 
     if (trackToEnable) {
         trackToEnable->setMode(TextTrack::Mode::Showing);
-
-        // If user preferences indicate we should always display captions, make sure we reflect the
```

Comment:
@eric-carlson - can you share your insights for following?

Reply:
I don't think it is an issue.